src/raftstore/store/peer_storage.rs

```diff
@@ -608,8 +608,7 @@ impl PeerStorage {
     #[inline]
     pub fn is_applying(&self) -> bool {
         match *self.snap_state.borrow() {
-            SnapState::Applying(_) |
-            SnapState::ApplyAborted => true,
```

Comment:
Why `ApplyAborted` consider as applying in before, for any other considerations?